src/encoded/static/components/award.js

```diff
@@ -1791,50 +1787,35 @@ export const ExperimentDate = (props) => {
     }
 
     let accumulatedDataReleased = [];
-    let accumulatedDataSubmitted = [];
     let date = [];
-    if (releasedDates.length > 0 || submittedDates.length > 0) {
+    if (releasedDates.length > 0) {
         const sortedreleasedTerms = consolidateSortedDates(sortTerms(releasedDates));
-        const sortedsubmittedTerms = consolidateSortedDates(sortTerms(submittedDates));
-
-        // Add an object with the most current date to one of the arrays.
-        if ((releasedDates && releasedDates.length > 0) && (submittedDates && submittedDates.length > 0)) {
-            if (sortedreleasedTerms.length > 0 && dayjs(sortedsubmittedTerms[sortedsubmittedTerms.length - 1].key).isAfter(sortedreleasedTerms[sortedreleasedTerms.length - 1].key, 'date')) {
-                sortedreleasedTerms.push({ key: sortedsubmittedTerms[sortedsubmittedTerms.length - 1].key, doc_count: 0 });
-            } else if (sortedsubmittedTerms.length > 0 && dayjs(sortedsubmittedTerms[sortedsubmittedTerms.length - 1].key).isBefore(sortedreleasedTerms[sortedreleasedTerms.length - 1].key, 'date')) {
-                sortedsubmittedTerms.push({ key: sortedreleasedTerms[sortedreleasedTerms.length - 1].key, doc_count: 0 });
-            }
-        }
 
         // Figure out the award start date. If none, use the earlier of the earliest released or submitted dates.
         let awardStartDate;
         if (award && award.start_date) {
             awardStartDate = dayjs(award.start_date).format('YYYY-MM');
         } else {
             const releasedIndex = sortedreleasedTerms.findIndex(item => item.doc_count);
-            const submittedIndex = sortedsubmittedTerms.findIndex(item => item.doc_count);
             const earliestReleased = releasedIndex > -1 ? sortedreleasedTerms[releasedIndex].key : sortedreleasedTerms[sortedreleasedTerms.length - 1];
-            const earliestSubmitted = submittedIndex > -1 ? sortedsubmittedTerms[submittedIndex].key : sortedsubmittedTerms[sortedsubmittedTerms.length - 1];
-            awardStartDate = earliestReleased < earliestSubmitted ? earliestReleased : earliestSubmitted;
+            awardStartDate = earliestReleased;
```

Comment:
You could just have: `const awardStartDate = releasedIndex...`

Reply:
👍